virttest/utils_sriov.py

```diff
@@ -99,6 +99,23 @@ def get_pf_pci(session=None):
             return pci_info.get('pci_id')
 
 
+def get_pf_info_by_pci(pci_id, session=None):
+    """
+    Get the pci info by the given pci id
+
+    :param pci_id: PF's pci id, eg. 0000:3b:00.0
+    :param session: The session object to the host
+    :return: Dict, pf's info,
+        eg. {'driver': 'ixgbe', 'pci_id': '0000:3b:00.0',
+             'iface': 'ens1f0', 'status': 'up'}
+    """
+    pf_info = get_pf_info(session=session)
+    for pf in pf_info.values():
+        if pf.get('pci_id') == pci_id:
+            logging.debug("PF %s details: %s.", pci_id, pf)
+            return pf
+
```

Comment:
Better explicitly return None if pci_id is not anticipated one

Reply:
I don't think so. It's redundant. May I ask why you want that?

Reply:
@Yingshun , perhaps it is my personal practice that what if you return sth in some function , we expect have consistent return value whatever condition  it meets. Maybe bring it to PR discussion for other opinoins.

Reply:
How about we agree to disagree? I'd prefer to keep the current way, but I wouldn't disagree with others if they explicitly return 'None' at the end of the function. 
BTW, is this a blocker for this function? If not, I think maybe we could continue to review. What do you think?